tests/openqa/webui/test_results.pm

```diff
@@ -57,6 +57,9 @@ sub run {
     for (1 .. 25) {
         send_key 'f5';
         wait_still_screen;
+        send_key 'home';
+        assert_and_click 'openqa-job-details';
+        wait_still_screen;
```

Comment:
This is a horrible waste of time :( I recommend you never add a `wait_still_screen` without argument. At best you can completely replace it with `assert_screen` or `assert_and_click` calls